packages/component/src/hooks/index.ts

```diff
@@ -18,17 +17,19 @@ import { useTypingIndicatorVisible } from '../BasicTypingIndicator';
 import { useSendBoxSpeechInterimsVisible } from '../SendBox/BasicSendBox';
 import { useMicrophoneButtonClick, useMicrophoneButtonDisabled } from '../SendBox/MicrophoneButton';
 import { useTextBoxSubmit, useTextBoxValue } from '../SendBox/TextBox';
+import { useRegisterFocusSendBox, type SendBoxFocusOptions } from './sendBoxFocus';
+
+export { type SendBoxFocusOptions };
 
 export {
   useDictateAbortable,
   useFocus,
-  /** @deprecated Please use `useFocus('sendBox')` instead. */
-  useFocusSendBox,
   useMakeThumbnail,
   useMicrophoneButtonClick,
   useMicrophoneButtonDisabled,
   useObserveScrollPosition,
   useObserveTranscriptFocus,
+  useRegisterFocusSendBox,
```

Comment:
Should we have a single `useRegisterFocus` and it can take both send box and chat history (and future stuff)?

Reply:
I think we can address it in the future. Not against, but not sure either.